web/packages/teleport/src/components/ServersideSearchPanel/ServersideSearchPanel.tsx

```diff
@@ -18,90 +18,41 @@
 
 import React from 'react';
 import { Flex } from 'design';
-import InputSearch from 'design/DataTable/InputSearch';
-import { PageIndicatorText } from 'design/DataTable/Pager/PageIndicatorText';
 
 import { AdvancedSearchToggle } from 'shared/components/AdvancedSearchToggle';
 
-import { PageIndicators } from 'teleport/components/hooks/useServersidePagination';
-
 import useServersideSearchPanel, {
-  HookProps,
   SearchPanelState,
-} from './useServerSideSearchPanel';
-
-interface ComponentProps {
-  pageIndicators: PageIndicators;
-  disabled?: boolean;
-}
-
-export interface Props extends HookProps, ComponentProps {
-  bigInputSize?: boolean;
-}
+  HookProps,
+} from 'teleport/components/ServersideSearchPanel/useServerSideSearchPanel';
 
-export default function Container(props: Props) {
-  const {
-    pageIndicators,
-    disabled,
-    bigInputSize = false,
-    ...hookProps
-  } = props;
-  const state = useServersideSearchPanel(hookProps);
-  return (
-    <ServersideSearchPanel
-      {...state}
-      pageIndicators={pageIndicators}
-      disabled={disabled}
-      bigInputSize={bigInputSize}
-    />
-  );
-}
+import { SearchInput } from './SearchInput';
 
-interface State extends SearchPanelState, ComponentProps {
-  bigInputSize?: boolean;
+export function ServersideSearchPanel(props: HookProps) {
+  const state = useServersideSearchPanel(props);
+  return <SearchPanel {...state} />;
 }
 
-export function ServersideSearchPanel({
+export function SearchPanel({
   searchString,
   setSearchString,
   isAdvancedSearch,
   setIsAdvancedSearch,
   onSubmitSearch,
-  pageIndicators,
-  disabled = false,
-  bigInputSize = false,
-}: State) {
+}: SearchPanelState) {
```

Comment:
I'm assuming that we got rid of `bigInputSize` since we don't use it, right?

Reply:
yes, it was always set to big